src/app/pages/signing/signing.component.html

```diff
@@ -77,9 +109,36 @@ <h5 class="modal-title" id="submit-confirm-modal-title">Are you sure you want to
         You can only submit one message to a friend’s virtrolio per account, so be absolutely certain!
       </div>
       <div class="modal-footer">
-        <button type="button" class="btn btn-primary" routerLink="/msg-sent" data-dismiss="modal">Submit</button>
+        <button type="button" class="btn btn-primary" (click)="createMsg(signingInputArea)" data-dismiss="modal">
+          Send it!</button>
         <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
       </div>
     </div>
   </div>
 </div>
+
+<!-- Information modal when they click on the ? -->
+<div class="modal fade" id="info-modal" tabindex="-1" role="dialog" aria-labelledby="info-modal-title"
+  aria-hidden="true">
+  <div class="modal-dialog modal-dialog-centered" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title" id="info-modal-title">Craft your perfect message!</h5>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body">
+        Virtrolio supports a modified version of <em>Markdown</em> to format text. We also use an emoji library! ❤️
+        <br>
+        <a href="https://www.markdownguide.org/basic-syntax/" target="_blank">Learn how to type in markdown</a>
+        <br>
+        <a href="https://github.com/ikatyang/emoji-cheat-sheet/blob/master/README.md" target="_blank">Visit the library
+          of available emojis</a>
+      </div>
+      <div class="modal-footer">
+        <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
+      </div>
+    </div>
+  </div>
+</div>
```

Comment:
No newline at end of file
```suggestion
</div>

```